src/boot/z_std_dma.c

```diff
@@ -229,7 +235,7 @@ void DmaMgr_DmaFromDriveRom(void* ram, uintptr_t rom, size_t size) {
  *
  * This function does not return.
  */
-NORETURN void DmaMgr_Error(DmaRequest* req, const char* file, const char* errorName, const char* errorDesc) {
+NORETURN void DmaMgr_Error(DmaRequest* req, const char* filename, const char* errorName, const char* errorDesc) {
     uintptr_t vrom = req->vromAddr;
```

Comment:
we use `file` way more than `filename`, at least currently. We can discuss fixing the inconsistency but it should prob be cleaned up in one pass
fwiw file is prob used more often for symmetry with the `__FILE__` directive

Reply:
ty